src/app/store/reducers/app-config.reducer.ts

```diff
@@ -24,10 +24,16 @@ export function reducer(
         loaded: true
       };
     }
+    case fromActions.LOGOUT: {
+      return {
+        ...state,
```

Comment:
maybe do to do or something because atm is not doing anything

Reply:
agree